site/src/components/mdx/mdx-page-content.tsx

```diff
@@ -177,7 +180,7 @@ export const MdxPageContent: FunctionComponent<MdxPageContentProps> = ({
             {
               marginTop: 0,
             },
-          "& > :first-child": {
+          "& > h1:first-of-type": {
```

Comment:
Fixes this warning

![Screenshot 2022-07-26 at 16 52 43](https://user-images.githubusercontent.com/16928688/181058755-0dee099d-1eea-4554-9ff6-7bb073405a65.png)